rollup.config.js

```diff
@@ -5,24 +5,32 @@ import resolve from 'rollup-plugin-node-resolve'
 
 const pkg = require('./package.json')
 
-export default {
-  input: 'src/index.js',
-  output: [
-    {
+export default [
+  {
+    input: 'src/index.js',
+    output: {
       file: pkg['module'],
       format: 'es'
     },
-    {
-      file: pkg['main'],
-      format: 'umd',
-      name: 'AutocompleteElement',
-      exports: 'named'
-    }
-  ],
-  plugins: [
-    resolve(),
-    babel({
-      presets: ['github']
-    })
-  ]
-}
+    external: '@github/combobox-nav',
+    plugins: [
+      resolve(),
+      babel({
+        presets: ['github']
+      })
+    ]
+  },
+  {
+    input: 'src/index.js',
+    output: {
+      file: pkg['browser'],
+      format: 'es'
+    },
+    plugins: [
+      resolve(),
+      babel({
+        presets: ['github']
+      })
+    ]
+  }
```

Comment:
> The browser build is required as browsers can't resolve `import '@github/combobox-nav'`.

The point of the `browser` field is to provide an alternative compilation that avoids using Node.js modules which cannot be resolved. As a more demonstrative example of this, the `browser` field can be an object that resolves various packages to alternate stub files; e.g. 

```json
{
  "browser": {
    "fs": "./fs-stub-browser.js"
  }
}
```

The expectation is that you're using a bundler which will rewrite `import` statements. External dependencies are fair game.

GitHub's compiler config respects the `browser` key, and so as this stands we would consume this browser version. We should drop this browser compile step, as it isn't needed.